### PR TITLE
Added policies endpoints wrapper

### PR DESCRIPTION
**Pull request by captaincrazybro**

### CODE REVIEW

File: src/data/policies.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+/**
+ *
+ * @description updates a current policy with the given 'policyData'
+ * @export
+ * @param {Number} id
+ * @param {Object} policyData
+ * @return {Object} 
+ */
+export async function updatePolicy(id, policyData) {
+    let parsedPolicyData = {
```
Comment (hobbitronics) [inline src/data/policies.js:21]: could this be a const?

File: src/data/policies.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        household_id: policyData.household_id,
+        cost_center: policyData.cost_center,
+        account: policyData.account,
+        entity_code: policyData.entity_code
+    }
+
+    let updatedPolicy = await PUT(`/policies/${id}`, parsedPolicyData)
```
Comment (hobbitronics) [inline src/data/policies.js:28]: same as above

**Review by hobbitronics — APPROVED** [review]
Looks good!

File: src/data/policies.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        household_id: policyData.household_id,
+        cost_center: policyData.cost_center,
+        account: policyData.account,
+        entity_code: policyData.entity_code
+    }
+
+    const updatedPolicy = await PUT(`/policies/${id}`, parsedPolicyData)
+
+    policies.update(currPolicies => {
+        let i = currPolicies.findIndex(pol => pol.id == id)
```
Comment (forevermatt) [inline src/data/policies.js:31]:
```suggestion
        let i = currPolicies.findIndex(pol => pol.id === id)
```

File: src/data/policies.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+        name: itemData.name,
+        category_id: itemData.category_id,
+        country: itemData.country,
+        description: itemData.description,
+        make: itemData.make,
+        model: itemData.model,
+        serial_number: itemData.serial_number,
+    }
+
+    const item = await POST(`/policies/${id}/items`, parsedItemData)
```
Comment (forevermatt) [inline src/data/policies.js:74]: After adding an item, you may want to reload the policies:
```suggestion
    const item = await POST(`/policies/${id}/items`, parsedItemData)
    loadPolicies()
```

Comment (captaincrazybro) [reply src/data/policies.js:74]: The GET `/policies` endpoint does not output any items property in each policy so I don't think that would really help any.

Comment (forevermatt) [reply src/data/policies.js:74]: Oh, sorry. You're right. I mixed up endpoints.

**Review by forevermatt — APPROVED** [review]

**Pull request merged**